src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -685,7 +673,7 @@ object DeepDiveLogCompiler extends DeepDiveLogHandler {
   val deepdiveVariableIdColumn = "dd_id"
   val deepdiveVariableLabelColumn = "dd_label"
   val deepdiveVariableLabelTruthinessColumn = "dd_truthiness"
-  val deepdivePrefixForVariablesIdsTable = "dd_variables_"
+  val deepdivePrefixForVariablesWithIdsTable = "dd_predicate_"
```

Comment:
Actually, could you elaborate a bit on the naming of this table?  "predicate" isn't very intuitive IMO. I think it's still a materialized variable, so maybe something straightforward as `dd_variable_copy_` or `dd_variable_with_id_`?